templates/media/media-gallery-item.php

```diff
@@ -29,9 +29,24 @@
 	<a href="<?php rtmedia_permalink(); ?>" title="<?php echo esc_attr( rtmedia_title() ); ?>"
 	   class="<?php echo esc_attr( apply_filters( 'rtmedia_gallery_list_item_a_class', 'rtmedia-list-item-a' ) ); ?>">
 
+		<?php
+			global $rtmedia_query;
+
+			$alt_text      = rtmedia_image_alt( false, false );
+			$rtmedia_media = $rtmedia_query->rtmedia;
+		?>
 		<div class="rtmedia-item-thumbnail">
-			<?php echo wp_kses( rtmedia_duration(), array( 'span' => array( 'class' => array() ) ) ); ?>
-			<img src="<?php rtmedia_image( 'rt_media_thumbnail' ); ?>" alt="<?php rtmedia_image_alt(); ?>">
+			<?php
+				echo wp_kses(
+					rtmedia_duration(),
+					array(
```

Comment:
@Sidsector9 for more readability you can store array in variable and then pass to function